Add 'now' variable to Snowfakery (#485)
SFDO-Tooling/Snowfakery: docs/index.md

will not chanage during the execution of
a single recipe.

#### `now`

The `now` variable returns a datetime

> representing the current moment. Experimentally,

Reviewer: Would it make sense to specifically mention that this returns the value of `datetime.now()`? I don't don't if the target audience is savvy enough to know what that means.

Author: I think not, in general. I'll see what Peter says.

#### `today`

The `today` variable returns a date
representing the current date. This date

> will not chanage during the execution of

Reviewer 2: ```suggestion
doesn't change during the execution of
```

a single recipe. See the `date` function
to learn more about this type of object.

#### `now`

The `now` variable returns a [datetime](#datetime)
representing the current moment. It outputs
microsecond precision and is in the UTC timezone.

Using this recipe, you can see three different
ways of outputting the timestamp:

```yaml
# tests/test_now.yml
- object: Times
  fields:
    current_datetime: ${{now}}
    current_datetime_as_number: ${{now.timestamp()}}
    current_datetime_without_microseconds: ${{int(now.timestamp())}}
```

> This would generate field values similar to these:

Reviewer 2: ```suggestion
The above example generates field values similar to these:
```
